src/Microsoft.AspNetCore.OData/Common/TypeHelper.cs

```diff
@@ -252,6 +252,26 @@ Type collectionInterface
             return false;
         }
 
+        /// <summary>
+        /// Checks if element is Primitive
+        /// </summary>
+        /// <param name="elementType"></param>
```

Comment:
Complete on the doc. What is the param, what is being returned?
```suggestion
        /// <summary>
        /// Check whether the given type is a primitive type or known type.
        /// </summary>
        /// <param name="type">The type to validate.</param>
        /// <returns>True if type is primitive or known type, otherwise False.</returns>
```